### DISCUSSION TITLE
Web Serial based terminal added

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#30] [Announcements]
Check out the new web terminal:

https://espresense.com/Web-Terminal/

**Comment by smmason** [comment]
Great feature!  Timestamps would be really handy for figuring out who's who.

**Reply by DTTerastar** [reply]
If you can try the version in the join branch, it's SUPER easy to figure it out.  Color coded and everything.  Could use a beta tester, I kinda changed everything in that branch.

**Reply by smmason** [reply]
I tried. The .bin I downloaded would not boot. 
That said, while I do know what I'm doing most of the time (30 year IT veteran, been flashing esp32 devices for a while now) as a non-coder, Github confuses me most of the time :) so if you could point me the proper file to use, I'll try again.

**Comment by DTTerastar** [comment]
I released it.  I need to make it easier for people to test a pre-release...  But anyways, now you should see a green line when a device comes near, and yellow when it goes further away after the green

**Reply by smmason** [reply]
Just got 1.7.1  Nice!
